src/Flicking.ts

```diff
@@ -581,194 +580,168 @@ class Flicking extends Component {
     return this.viewport.remove(index, deleteCount);
   }
 
+  /**
+   * Mapping all items of the user for the currently visible panels.
+   * @private
+   * @ko 현재 보이는 패널들에 대해 사용자의 전체 아이템들과 매핑을 시킨다.
+   * @param - User's all items <ko>사용자의 전체 아이템들</ko>
+   * @return Array of Item of the user mapped to the currently visible item <ko>현재 보이는 패널과 매핑되는 사용자의 아이템 배열</ko>
```

Comment:
Trivial, but I think a comma would be better than blank space after `items` and `item`